lib/write-i18n-loaders.js

```diff
@@ -28,7 +28,10 @@ const createTranslationLoader = (loaderName, locale, format) => {
     return (
       `const ${loaderName} = (callback, scope) =>
    import( /* webpackChunkName: "${locale}-translations" */ '${locale}.js')
-     .then((module) => { callback.call(scope, module);})
+     .then((module) => { 
+       callback.call(scope, module);
+       return Promise.resolve(module);
```

Comment:
do we want to return the promised value since it wasn't before?

Reply:
Good point